modules/xml/src/main/resources/META-INF/mule-xml.xsd

```diff
@@ -514,14 +514,25 @@ The Jaxen filter allows you to route messages based on XPath expressions. The Ja
                     </xsd:element>
                     <xsd:element name="context-property" type="spring:entryType" minOccurs="0" maxOccurs="unbounded">
                         <xsd:annotation>
-                            <xsd:documentation>A property that wil be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
+                            <xsd:documentation>A property that will be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
                             </xsd:documentation>
                         </xsd:annotation>
                     </xsd:element>
                 </xsd:sequence>
+                <xsd:attribute name="timeBetweenTransformerEvictions" type="mule:substitutableInt">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                          Transformers are pooled for better throughput, since performing an XSL
```

Comment:
To the end of my previous comment, I've added `XsltTransformerPoolEvictionTestCase`. The single test in that class ensures proper eviction functionality. There was no need to change production code to get idle object eviction.

Note that I greatly despise the state of `XsltTransformerPoolEvictionTestCase` since it's difficult to digest. I've attempted to make it as straight-forward as possible using appropriate method factorization and explanatory comments. I believe it's ugliness is due mainly to the fact that it tests a necessarily mutithreaded scenario, but any and all improvements are very welcome.